app/src/main/java/com/haruta/harutyan/countchallenge/MainActivity.kt

```diff
@@ -21,10 +23,20 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater).apply { setContentView(this.root) }
 
+        var saveNumber: SharedPreferences = getSharedPreferences("SharedPref", Context.MODE_PRIVATE)
+
+        var str = saveNumber.getInt("COUNT", countNumber)
```

Comment:
`str` だと何の値が変数に入るか分かりにくくなってしまうので、一時的に使う変数でも命名はこだわると良いぜ！

```suggestion
        var savedCount = saveNumber.getInt("COUNT", countNumber)
```